test/services/BusinessServiceSpec.scala

```diff
@@ -41,13 +42,13 @@ class BusinessServiceSpec extends AnyWordSpecLike {
   val ifsBusinessDetailsConnector: StubIFSBusinessDetailsConnector = StubIFSBusinessDetailsConnector()
   val mdtpConnector: StubMDTPConnector                             = StubMDTPConnector()
   val ifsConnector: StubIFSConnector                               = StubIFSConnector()
-  val service                                                      = new BusinessServiceImpl(ifsBusinessDetailsConnector, mdtpConnector, ifsConnector)
+  val testService                                                  = new BusinessServiceImpl(ifsBusinessDetailsConnector, mdtpConnector, ifsConnector)
```

Comment:
This is to differentiate between this service and the one used in the Test trait at the bottom of the classs